README.md

```diff
@@ -14,9 +14,10 @@ We support multiple data types for flags (numbers, strings, booleans, objects) a
 
 | Ruby Version | OS                          |
 | -----------  | -----------                 |
-| Ruby 2.7.6   | Windows, MacOS, Linux       |
-| Ruby 3.0.4   | Windows, MacOS, Linux       |
-| Ruby 3.1.2   | Windows, MacOS, Linux       |
+| Ruby 2.7.8   | Windows, MacOS, Linux       |
+| Ruby 3.0.6   | Windows, MacOS, Linux       |
+| Ruby 3.1.4   | Windows, MacOS, Linux       |
+| Ruby 3.2.2   | Windows, MacOS, Linux       |
```

Comment:
I did this in the beginning, and the thought is to add as needed

Reply:
I would either add tests for MacOS and Windows or remove the OS column from the readme. We should have access to the mac and windows GitHub action runners in this account, but I think this would be the first repo in the org to use them.

@josecolella what do you think?

Reply:
@maxveldink, I've enabled auto-merge on this PR once this thread has been resolved.

Reply:
@maxveldink Going to merge this and I can follow with the removal of the column